homeassistant/components/fan/__init__.py

```diff
@@ -45,6 +45,7 @@
 SPEED_OFF = 'off'
 SPEED_LOW = 'low'
 SPEED_MEDIUM = 'medium'
+SPEED_MED = 'med'
```

Comment:
So you'd prefer to have everything with in there with `SPEED_OFF`, `SPEED_LOW`,` 'med'`, `SPEED_HIGH` ?

If that's the case, @robbiet480, you could have taken her new information - the part you were missing - and completed your improvements however it is best coded.  But it's not complete how it is.  It still doesn't work.  It just discovers again.   You can see the inconsistency in the code.  ISY doesn't understand "medium"  With #5817 applied, a fan in switching from low to medium shuts off.  A fan that's off switched to medium does nothing.


I'd disagree with your collective decision that all "Fans use SPEED_MEDIUM".  Clearly this one doesn't.  Perhaps there should be a comment to the right of SPEED_MED in `init` that says:
```python
SPEED_MED = 'med' #used for ISY994
```
and any new exception that creeps in gets the same treatment.  Or even move it to the end of the preferred list.  But aesthetically, I think that's better than looking at a `SPEED_OFF`, `SPEED_LOW`, `'med'`, `SPEED_HIGH`.  But... you know, what do I know?

Still, I could drop the `init` change and change all the references inside `isy994` to `'med'` or you could come to terms with the constant that's been there for awhile already.  I prefer the first.  It sounds like you guys prefer the second.  Just let me know what you'd like.  Or if you want to finish it out Robbie, let me know that and I'll withdraw the PR with no hard feelings.  I'm just hoping to not have to patch the releases to make the house work.  But I am glad it is working again, one way or the other.

I really do appreciate all the work you guys put into this.